theforeman.org/pipelines/lib/rvm.groovy

```diff
@@ -13,9 +13,13 @@ def gemset(name = null) {
     base_name
 }
 
-def configureRVM(ruby, name = '') {
+def configureRVM(ruby, name = '', bundler_version = null) {
     emptyGemset(ruby, name)
-    withRVM(["gem install bundler --no-document"], ruby, name)
+    if (bundler_version) {
+        withRVM(["gem install bundler -v '${bundler_version}' --no-document"], ruby, name)
+    } else {
+        withRVM(["gem install bundler --no-document"], ruby, name)
+    }
```

Comment:
the alternative would be to fall back `bundler_version` to something like `>= 0`, but I don't mind either way

Reply:
I thought this was easier to read in the job output